BlockSettleUILib/AuthAddressViewModel.cpp

```diff
@@ -256,18 +256,9 @@ bool AuthAdressControlProxyModel::isUnsubmittedAddressVisible() const
    return false;
 }
 
-void AuthAdressControlProxyModel::adjustVisibleCount()
-{
-   for (; visibleRowsCount_ < sourceModel_->rowCount(); ++visibleRowsCount_) {
-      if (sourceModel_->isAddressNotSubmitted(visibleRowsCount_)) {
-         break;
-      }
-   }
-
-   invalidate();
-}
-
 bool AuthAdressControlProxyModel::filterAcceptsRow(int row, const QModelIndex&) const
 {
+   const int max_rot = visibleRowsCount_;
```

Comment:
typo? or I don't get what is "rot" here.
Also, looks like it's not used further.

Reply:
debug code. removed.